solver/llbsolver/solver.go

```diff
@@ -242,7 +242,7 @@ func (s *Solver) Solve(ctx context.Context, id string, sessionID string, req fro
 			inp.Refs = m
 		}
 		if _, ok := asInlineCache(exp.CacheExporter); ok {
-			if err := inBuilderContext(ctx, j, "preparing layers for inline cache", "", func(ctx context.Context, _ session.Group) error {
+			if err := inBuilderContext(ctx, j, "preparing layers for inline cache", j.SessionID+e.ID(), func(ctx context.Context, _ session.Group) error {
```

Comment:
Not sure I understand this as isn't the `SessionID` already unique for separate bake targets in current implementation.

As this is only used for digest we should be able to just generate a unique id in here and not bother with the `ID()` function?